functions/brinks/src/lib/generateAllocationsFromOrder.js

```diff
@@ -0,0 +1,31 @@
+import debug from 'debug';
+
+const dlog = debug('that:api:functions:brinks:generateAllocationsFromOrder');
+
+export default function generateAllocationsFromOrder({
+  order,
+  orderCreatedAt,
+}) {
+  dlog('generateAllocationsFromOrder called');
+
+  const { lineItems } = order;
+  const orderAllocations = [];
+
+  lineItems.forEach(lineItem => {
```

Comment:
map would be faster and potentially cleaner.

Reply:
I am pretty sure they run the same when running over all elements. I see how this can be refactored to a map function returning the value `orderAllocations` over pushing a new array value on success. If it is preferred please say so and I will refactor.

Reply:
You right that map can be a bit faster. It does make more sense in this application so I will refactor this function using `map()` instead.

Reply:
@csell5 I went to refactor this and actually `forEach` is the correct approach for this scenario. The resulting array will most probably be larger than the array of line items. In this function we are creating 1 allocation row for each number of quantity. So if there is 1 line items with a quantity of 3 we'll end up with an array with three elements, not 1 which is what you end up with using map. I don't see how we can utilize the the return value of `map()` for this and imo this seems to be the better approach.